### PR TITLE
k2tf: init at 0.5.0

### PR DESCRIPTION
**Pull request by flokli**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
Result of `nixpkgs-review pr 114359` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>k2tf</li>
  </ul>
</details>

File: pkgs/development/tools/misc/k2tf/default.nix [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  src = fetchFromGitHub {
+    owner = "sl1pm4t";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0i1bhn0sccvnqbd4kv2xgng5r68adhcc61im2mn8hxmds5nf6in2";
+  };
+
+  vendorSha256 = "1c2mwhrj0xapc661z1nb6am4qq3rd1pvbvjaxikjyx95n0gs8gjk";
+
```
Comment (fabaff) [inline pkgs/development/tools/misc/k2tf/default.nix:15]: Could you please check if you could set the version?

```bash
buildFlagsArray+=("-ldflags" "-s" "-w" "-X whatever=v${version}")
```

**Review by fabaff — COMMENTED** [review]
- [x] package path fits guidelines
- [x] package name fits guidelines
- [x] package version fits guidelines
- [x] `meta.description` is set and fits guidelines
- [x] `meta.license` fits upstream license
- [x] `meta.maintainers` is set
- [x] package built (see https://github.com/NixOS/nixpkgs/pull/114359#issuecomment-785894986)
- [ ] all depending packages build
- [x] tested on x86_64

```bash
...ew/pr-114359]$ ./results/k2tf/bin/k2tf -h
Usage of ./results/k2tf/bin/k2tf:
  -d, --debug                 enable debug output
[...]
ew/pr-114359]$ ./results/k2tf/bin/k2tf -v
k2tf version: dev
```

Looks good to me besides the missing version. But that is not a blocker.

File: pkgs/development/tools/misc/k2tf/default.nix (hunk 1 above)
Comment (flokli) [reply pkgs/development/tools/misc/k2tf/default.nix:15]: followup in https://github.com/NixOS/nixpkgs/pull/114368

**Pull request merged**